packages/ndla-ui/src/locale/messages-nn.ts

```diff
@@ -1297,7 +1297,14 @@ const messages = {
         term2: 'Ikkje skriv noko støytande i tekstfelt.',
         term3: 'NDLA tek atterhald om retten til å oppdatere eller slette utdaterte ressursar.',
       },
-      newFavourite: 'Nyleg lagt til',
+      recentFavourites: {
+        title: 'Nyleg til i mine favorittar',
+        link: 'Sjå alle mappana dine',
```

Comment:
Er ikkje a i alle nynorske ord.
```suggestion
        link: 'Sjå alle mappene dine',
```